src/components/SideBar/AppPicker/index.tsx

```diff
@@ -0,0 +1,85 @@
+import {
+  Box as MUIBox,
+  ButtonBase as MUIButton,
+  StylesProvider,
+} from '@material-ui/core';
+import React from 'react';
+import { connect } from 'react-redux';
+import { AppAction } from 'redux/actionTypes';
+import { AppState } from 'redux/reducer';
+import styled from 'styled-components';
+import theme from 'styles/theme';
+
+export const AppPickerComponent: React.FunctionComponent = () => {
+  return (
+    <StylesProvider injectFirst>
```

Comment:
what is this for?

Reply:
im not sure what it does but 
without it
this happens
![image](https://user-images.githubusercontent.com/20599510/81431244-d8cc1d00-912e-11ea-8706-8fb995b606e0.png)

Reply:
ahh i see. the reason thats happening is b/c you're using the `styled` function from `styled-components` to style MaterialUI components. if you want to style material UI components you should use `import { styled } from '@material-ui/core/styles';`

Reply:
lol that low key breaks a lot of the styling code / we also style normal `div` and `img` etc. -- should we be importing & using both?

Reply:
yeah you can use both. i usually do:
```
import { styled as muiStyled } from '@material-ui/core/styles'
import styled from "styled-components:"
```